tests/test_core.py

```diff
@@ -144,7 +144,10 @@ def test_geodataframe_crs(geodf_points_crs):
     assert dask_obj.partitions[1].crs == original
 
     new_crs = "epsg:4316"
-    new = dask_obj.set_crs("epsg:4316")
+    with pytest.raises(ValueError):
```

Comment:
Can we add a 'match=` here? To ensure that we capture the correct error. Same below.

Reply:
Added a match with `.*already has a CRS which is not equal to the passed CRS.*`